src/fspp/fuse/Fuse.cpp

```diff
@@ -235,7 +235,7 @@ void Fuse::run(const bf::path &mountdir, const vector<string> &fuseOptions) {
   ASSERT(_argv.size() == 0, "Filesystem already started");
 
   _argv.reserve(2 + fuseOptions.size());
-  _argv.push_back(_create_c_string("fspp")); // The first argument is the executable name
+  _argv.push_back(_create_c_string("cryfs")); // The first argument is the executable name
```

Comment:
The fspp library is meant to be an abstraction layer independent from CryFS, so the cryfs name shouldn't be hardcoded here. You can, however, pass a file system name in to the Fuse::Fuse() constructor, store it as a member variable and then use it here.